### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by alnicoli**

### CODE REVIEW

File: packages/verification-vue-component/package.json [hunk 1]
```diff
@@ -41,31 +41,31 @@
...
         "eslint-plugin-standard": "^4.1.0",
-        "eslint-plugin-vue": "^7.20.0",
+        "eslint-plugin-vue": "^9.8.0",
         "glob": "^8.0.3",
         "npm-run-all": "^4.1.5",
-        "postcss": "^8.4.16",
+        "postcss": "^8.4.20",
         "postcss-cli": "^9.1.0",
         "rimraf": "^3.0.2",
         "rollup": "^2.79.0",
```
Comment (bauer00) [inline packages/verification-vue-component/package.json:64]: Version not consistent over packages

File: packages/verification-app/package.json [hunk 2]
```diff
@@ -3,41 +3,42 @@
...
+        "eslint": "^8.28.0",
+        "eslint-config-standard": "^17.0.0",
         "eslint-plugin-import": "^2.26.0",
         "eslint-plugin-node": "^11.1.0",
-        "eslint-plugin-promise": "^4.3.1",
+        "eslint-plugin-promise": "^6.1.1",
         "eslint-plugin-standard": "^4.1.0",
-        "eslint-plugin-vue": "^7.20.0",
+        "eslint-plugin-vue": "^9.8.0",
         "sass": "^1.54.9",
```
Comment (bauer00) [inline packages/verification-app/package.json:35]: Version not consistent over packages

File: packages/verification-app/package.json [hunk 3]
```diff
@@ -3,41 +3,42 @@
...
         "lint": "vue-cli-service lint",
         "build:dev": "vue-cli-service build --mode development",
         "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'"
     },
     "dependencies": {
         "@certifaction/verification-vue-component": "^3.5.0",
         "axios": "^0.27.2",
-        "bootstrap": "^5.2.1",
+        "bootstrap": "^5.2.3",
         "core-js": "^3.25.2",
```
Comment (bauer00) [inline packages/verification-app/package.json:16]: Should be no problem to update to latest version

File: packages/verification-app/package.json [hunk 4]
```diff
@@ -3,41 +3,42 @@
...
-        "@vue/cli-service": "~4.5.19",
+        "@vue/cli-plugin-babel": "~5.0.8",
+        "@vue/cli-plugin-eslint": "~5.0.8",
+        "@vue/cli-service": "~5.0.8",
         "@vue/eslint-config-standard": "^5.1.2",
-        "babel-eslint": "^10.1.0",
+        "@babel/eslint-parser": "^7.19.1",
         "cross-env": "^7.0.3",
-        "eslint": "^6.8.0",
+        "eslint": "^8.28.0",
```
Comment (bauer00) [inline packages/verification-app/package.json:28]: Is 8.30.0 not working?

File: packages/verification-vue-component/package.json [hunk 5]
```diff
@@ -41,31 +41,31 @@
         "vue-i18n": "^8.27"
     },
     "devDependencies": {
-        "@babel/core": "^7.19.1",
-        "@babel/preset-env": "^7.19.1",
+        "@babel/core": "^7.20.5",
```
Comment (bauer00) [inline packages/verification-vue-component/package.json:44]: Not latest version

File: packages/verification-vue-component/package.json [hunk 6]
```diff
@@ -41,31 +41,31 @@
...
+        "@babel/preset-env": "^7.19.6",
```
Comment (bauer00) [inline packages/verification-vue-component/package.json:45]: 7.20 doesn't work?

File: yarn.lock [hunk 7]
```diff
@@ -1,16164 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (bauer00) [inline yarn.lock:1]: I think something went wrong here 😅

Comment (alnicoli) [reply yarn.lock]: No idea how it got removed but readded now :D Not my day

**Review by bauer00 — APPROVED** [review]

**Pull request merged**